src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -62,9 +64,14 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         Set<Appointment> appointmentList = Collections.emptySet();
+        MedCon medCon = argMultimap.getValue(PREFIX_MEDCON).isPresent()
+                ? ParserUtil.parseMedCon(argMultimap.getValue(PREFIX_MEDCON).get())
+                : new MedCon("");
```

Comment:
As Medical Condition is not part of the parameters for `AddCommand`, you can initialise `medCon` to be an empty string